server/opts.go

```diff
@@ -1685,7 +1690,7 @@ func (o *Options) processConfigFileLine(k string, v any, errors *[]error, warnin
 	case "reconnect_error_reports":
 		o.ReconnectErrorReports = int(v.(int64))
 	case "websocket", "ws":
-		if err := parseWebsocket(tk, o, errors); err != nil {
+		if err := parseWebsocket(tk, o, errors, warnings); err != nil {
```

Comment:
So we got this warning populated in the parseWebsocket (wired interface, btw), but we never used it.
Looks like we need to print them in 
https://github.com/nats-io/nats-server/blob/e3dcbfd7c2655ac598697871416d8ca082e62d84/server/opts.go#L864 ?